test/exercism/homework_test.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../test_helper'
+require_relative '../../lib/exercism'
+require_relative '../../lib/exercism/homework'
```

Comment:
So MiniTest wasn't all that difficult to figure out, although for some reason I was getting an `NameError: uninitialized constant HomeworkTest::Homework` error until I explicitly required this file. I though it would have been included in the `require_relative '../../lib/exercism'` call above, but I guess it wasn't for some reason?